src/test_workflow/integ_test/integ_test_suite_opensearch_dashboards.py

```diff
@@ -57,10 +58,13 @@ def __init__(
         )
 
         # Integ-tests for OSD now clones FunctionalTestDashboards Repository by default and points to integtest.sh from FunctionalTestDashboards for all OSD plugins
+        functional_test_dashboards_ref = ft_repo_ref
```

Comment:
Suggest getting consistent on the naming:
1. If it is `functional_test_repo`, then use it consistently and avoid `ft_repo`.
2. Keep it consistent with `opensearch_dashboards` instead of `dashboards`. (ex: build_manifest_opensearch_dashboards)

Reply:
@peterzhuamazon just saw these comments. Will work on it soon.